internal/controller/cloud-resources/cr/doc.go

```diff
@@ -0,0 +1,7 @@
+package cr
+
+/*
+The CloudResources reconciler must be tested in islation since then CloudResource CR is deleted
```

Comment:
nitpicking:
typo: "isolation"
Also the sentence here with 2 "then"s is not readable.